### DISCUSSION TITLE
希望插件能支持在线TTS

### DISCUSSION THREAD

**Discussion by Zawinzala** [xfgryujk/blivechat#231] [想法]
比如百度，腾讯，阿里云，或者是Openai。

理由：内置的TTS还是效果太差。效果好的大模型TTS都是云服务类型。比如我喜欢的openai。

希望能支持在线api的调用：

这里附上openai调用的文档：

https://platform.openai.com/docs/api-reference/audio

其实隔壁的
![Bililive_dm_jLUPqnXjSr](https://github.com/user-attachments/assets/f1d12db9-b8d7-4e76-aff3-d48f9d3fcfd3)
有这个功能，但是好几年没更新，在线tts的调用有bug，只能实现单纯的get请求，post请求会有问题。

**Comment by xfgryujk** [comment]
你不会自己做吗

**Comment by Zawinzala** [comment]
确实不会，试着看了一下不太明白。我比较菜
